tardis/plasma/tests/test_complete_plasmas.py

```diff
@@ -1 +1,139 @@
+import os
 
+import pandas as pd
+import pytest
+from pandas.util import testing as pdt
+from numpy.testing import assert_almost_equal
+from tardis.io.config_reader import Configuration
+from tardis.simulation import Simulation
+
+
+setupI = {
```

Comment:
We don't need to test all permutations. That would be too much reference data. But we can test all options 'alone'. I think this would give us ~20 different setups which I think is doable.